vnext/Microsoft.ReactNative/Fabric/Composition/CompositionContextHelper.cpp

```diff
@@ -622,9 +622,9 @@ struct CompActivityVisual : winrt::implements<
     auto compositor = m_visual.Compositor();
```

Comment:
Is there any reason why you can't just call the `updateSize()` rather than having a duplicate of the code?

Reply:
updateSize also clears the animations and spriteVisuals which isn't necessary when first creating the activityIndicator! But those are minor and it may be cleaner to just remove the duplicate code :)